src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -408,16 +408,26 @@ public string PackageDeprecationAlternatePackageText
             }
         }
 
-        public bool IsPackageVulnerable
+        private IReadOnlyCollection<PackageVulnerabilityMetadataContextInfo> _packageVulnerabilities;
+        public IReadOnlyCollection<PackageVulnerabilityMetadataContextInfo> PackageVulnerabilities
         {
-            get { return PackageVulnerabilityMaxSeverity > -1; }
+            get { return _packageVulnerabilities; }
```

Comment:
```suggestion
            get => _packageVulnerabilities;
```

Reply:
Updated locally. Changes will be reflected with next commit / push.